docs/Setup/NodeSDK-setup.md

```diff
@@ -0,0 +1,253 @@
+
+```
+go get -u github.com/kardianos/govendor
+cd $GOPATH/src/github.com/chaincode_example02
+govendor init
+govendor fetch github.com/hyperledger/fabric
+```
+
+Now, execute `go build` to verify that all of the chaincode dependencies are present.
+
+```
+go build
+```
+
+Next, we will switch over to the node sdk directory in the fabric repo to run the node sdk tests, to make sure you have everything properly set up. Verify that the [chain-tests.js](https://github.com/hyperledger/fabric/blob/master/sdk/node/test/unit/chain-tests.js) unit test file points to the correct chaincode project path. The default directory is set to `github.com/chaincode_example02/` as shown below. If you placed the sample chaincode elsewhere, then you will need to change that.
+
+```
+// Path to the local directory containing the chaincode project under $GOPATH
+var testChaincodePath = "github.com/chaincode_example02/";
+```
+
+**Note:** You will need to run `npm install` the first time you run the sdk tests, to install all of the dependencies. Set the `DEPLOY_MODE` environment variable to `net` and run the chain-tests as follows:
+
+```
+cd $GOPATH/src/github.com/hyperledger/fabric/sdk/node
+npm install
+export DEPLOY_MODE='net'
+node test/unit/chain-tests.js | node_modules/.bin/tap-spec
+```
+
+### Enabling TLS
+
+If you wish to activate TLS connection with the member services the following actions are needed:
+
+- Modify `$GOPATH/src/github.com/hyperledger/fabric/membersrvc/membersrvc.yaml` as follows:
+
+```
+server:
+     tls:
+        cert:
+            file: "/var/hyperledger/production/.membersrvc/tlsca.cert"
+        key:
+            file: "/var/hyperledger/production/.membersrvc/tlsca.priv"
+```
+
+This is needed to instruct the member services on which tls cert and key to use.
+
+- Modify `$GOPATH/src/github.com/hyperledger/fabric/peer/core.yaml` as follows:
+
+```
+peer:
+    pki:
+        tls:
+            enabled: true
+            rootcert:
+                file: "/var/hyperledger/production/.membersrvc/tlsca.cert"
+```
+
+This is needed to allow the peer to connect to the member services using TLS, otherwise the connection will fail.
+
+- Bootstrap your member services and the peer. This is needed in order to have the file *tlsca.cert* generated by the member services.
+
+- Copy `/var/hyperledger/production/.membersrvc/tlsca.cert` to `$GOPATH/src/github.com/hyperledger/fabric/sdk/node`.
+
+*Note:* If you cleanup the folder `/var/hyperledger/production` then don't forget to copy again the *tlsca.cert* file as described above.
+
+## Running Unit Tests
+HLC includes a set of unit tests implemented with the [tape framework](https://github.com/substack/tape). The unit [test script](https://github.com/hyperledger/fabric/blob/master/sdk/node/bin/run-unit-tests.sh) builds and runs both the membership service server and the peer node for you, therefore you do not have to start those manually.
```

Comment:
membership services >>Membership Services

The unit [test script] >> The [unit test script]